app/controllers/posts_controller.rb

```diff
@@ -3,6 +3,10 @@ class PostsController < ApplicationController
 
   def index
     @posts = Post.recent.published.includes(:tags)
+    respond_to do |format|
+      format.html
+      format.json { render json: @posts }
+    end
```

Comment:
it's fine if you don't want to waste more time on this, but wouldn't it be better to have a different endpoint? not only to avoid this `respond_to` block, but also to only return the last two posts, instead of the last 10, which is a waste?